base/applications/regedit/lang/ro-RO.rc

```diff
@@ -32,16 +32,16 @@ Opțiuni:\n\
   /R:user.dat    Locația fișierului user.dat urmează să fie modificată.\n\
                  Compatibil cu orice alt comutator. Ignorat.\n\
   /C             Importă conținutul unui fișier de regiștri.\n\
-  /D             Șterge o cheie de regiștrii specificată.\n\
-  /E             Exportă conținutul unei cheie de regiștri specificată într-un fișier.\n\
+  /D             Șterge o cheie de registru specificată.\n\
+  /E             Exportă conținutul unei chei de registru specificată într-un fișier.\n\
                  Dacă nu este nicio cheie specificată, toți regiștrii sunt exportați.\n\
```

Comment:
I suggest to include this instance too (i.e. "este exportat întregul registru").

Reply:
The comment written a while ago was meant for this line only (which you left unchanged). Since it missed the target, let me be a bit more specific then - one suggested option for fixing "toți regiștrii sunt exportați" was to replace it with "este exportat întregul registru".